RUNTIME.md

```diff
@@ -0,0 +1,536 @@
+
+
+
+
+
+The GEOPM package provides many built-in features.  An advanced use
+case is dynamically coordinating hardware settings across all compute
+nodes used by an distributed HPC application in response to the
+application's behavior and resource manager requests.  The dynamic
+coordination is implemented as a hierarchical control system for
+scalable communication and decentralized control. The hierarchical
+control system can optimize for various objective functions including
+maximizing global application performance within a power bound or
+minimizing energy consumption with marginal degradation of application
+performance.  The root of the control hierarchy tree can communicate
+with the system resource manager to extend the hierarchy above the
+individual MPI application and enable the management of system power
+resources for multiple MPI jobs and multiple users by the system
+resource manager.
+
+The GEOPM package provides two libraries: libgeopm for use with MPI
```

Comment:
What about libgeopmd?  There's no discussion of the service here.

Reply:
I've been trying to refer to the "GEOPM package" and the "GEOPM Service package" perhaps we should call the base build package the "GEOPM HPC Runtime package" throughout?